packages/pigeon/pigeons/core_tests.dart

```diff
@@ -48,6 +48,7 @@ class AllTypes {
 }
 
 /// A class containing all supported nullable types.
+@SwiftClass()
```

Comment:
should we test both class and struct cases?

Reply:
yes, but that will delay this pr from landing as the testing is extensive. I'm already working on it though.